api/tests/Behat/features-v2/registration/ingest.lay.sirius.feature

```diff
@@ -1,4 +1,4 @@
-@v2 @registration @ingest @v2_sequential
+@v2 @registration @ingest @v2_sequential 
```

Comment:
Seems like this file hasn't changed expect for adding some white-space. Maybe worth reverting?